pkg/cmd/kind/load/docker-image/docker-image.go

```diff
@@ -42,15 +42,15 @@ type flagpole struct {
 func NewCommand() *cobra.Command {
 	flags := &flagpole{}
 	cmd := &cobra.Command{
-		Args: func(cmd *cobra.Command, args []string) error {
```

Comment:
This should have been valid, cobra uses this semantically to validate arguments

https://github.com/spf13/cobra/blob/77e4d5aecc4d34e58f72e5a1c4a5a13ef55e6f44/args.go#L7

Reply:
@BenTheElder I tested switching from `PreRunE` to `Args` and I cannot see logs. Give it a try on your own: doesn't work because it looks `Args` is executed &before* the persistent preRun :(

Reply:
As you can see [here](https://github.com/spf13/cobra/blob/77e4d5aecc4d34e58f72e5a1c4a5a13ef55e6f44/command.go#L802-L804), `Args` validation is performed before `PersistentPreRunE`

Reply:
this problem applies to other subcommands, we need a more general solution

Reply:
Nope, Kind is just using the `Args` func for `load docker-images` and `load image-archive` sub-commands.

The unique exception is the `export logs` but `Args` there is optional:
https://github.com/kubernetes-sigs/kind/blob/aea9a47507085fe7bf89f7f65d31ead92876a42d/pkg/cmd/kind/export/export.go#L29-L30

Reply:
`NoArgs` returns an error when there are unexpected arguments.

Cobra can also error in pflag parsing. We need an approach that can handle these. I think I've found that in https://github.com/kubernetes-sigs/kind/pull/1050